### PR TITLE
Migrate SQS Connector to Ballerinax on Ballerina 1.2.0

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
Migrate SQS Connector to Ballerina with path _ballerinax/aws.sqs_.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,42 +24,42 @@ git clone https://github.com/wso2-ballerina/module-amazonsqs.git
...
 ```
 
 In order for you to use the Amazon SQS Connector, first you need to create an Amazon SQS Client.
 
-Ballerina provides a [config module](https://ballerina.io/learn/api-docs/ballerina/config.html) to obtain parameters from the configuration file. Specify the configuration object and create the client as follows.
+Ballerina provides a [config module](https://ballerina.io/v1-1/learn/by-example/config-api.html) to obtain parameters from the configuration file. Specify the configuration object and create the client as follows.
 
 ```ballerina
-amazonsqs:Configuration configuration = {
+aws.sqs:Configuration configuration = {
```
Comment (chamil321) [inline README.md]: > "Module names can contain alphanumeric characters including dots (.). Dots in a module name has no meaning other than the last segment after the final dot being used as a default alias within your source code."
```suggestion
sqs:Configuration configuration = {
```

Comment (chamil321) [reply README.md]: As per the docs, we can just use the final part for the module name. So shall we change it accordingly?

Comment (Maninda) [reply README.md]: Changed in all locations.

File: .travis.yml [hunk 2]
```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 script:
-  - wget https://product-dist.ballerina.io/downloads/1.1.0/ballerina-linux-installer-x64-1.1.0.deb
-  - sudo dpkg -i ballerina-linux-installer-x64-1.1.0.deb
+  - wget https://product-dist.ballerina.io/downloads/1.1.4/ballerina-linux-installer-x64-1.1.4.deb
```
Comment (niveathika) [inline .travis.yml]: Shouldn't this be 1.2.0?

Comment (Maninda) [reply .travis.yml]: Updated.

**Pull request merged**